docs/index.mustache.html

```diff
@@ -23,6 +23,7 @@
             : 'https://vim.azureedge.net/samples/residence.vim';
 
         var viewer = new vim.Viewer({
+            mouseOrbit: true,
```

Comment:
I would suggest make this false, until mouseOrbit supports pan, but wait for Luke Hamilton or Joel's suggestion.